src/main/java/approximations/java/lang/StringBuffer_Tests.java

```diff
@@ -1035,7 +1033,9 @@ public static int test_deleteCharAt_0(final int execution) {
 
 
     // StringBufferAutomaton::ensureCapacity (int)
+    @Test(disabled = true)
```

Comment:
Using `DO_NOTHING` action within the approximation does not necessarily mean that this method should not be tested/called at all.

I suggest keeping this test *enabled* and organizing the test in question around making calls to `ensureCapacity` (when a `StringBuffer` is *empty or not*) but **without checking the side-effect directly**: we are expecting it to not throw any exceptions and not affect the contents in any way. You can check the `length()` to "make sure the buffer hasn't lost/gain its contents".

This also applies to the `trimToSize()` method.